SystemServices/SystemServices.cpp

```diff
@@ -243,6 +245,8 @@ namespace WPEFramework {
             : AbstractPlugin()
               , m_cacheService(SYSTEM_SERVICE_SETTINGS_FILE)
         {
+            Core::JSONRPC::Handler& systemVersion_2 = JSONRPC::CreateHandler({ 2 }, *this);
```

Comment:
Pls use the new registerMethod() API with versioning support and pass in the versions. FYI this is the change done in 2010 sprint - https://github.com/rdkcentral/rdkservices/pull/535/files

Reply:
Used new new registerMethod().